tests/utils.py

```diff
@@ -335,6 +335,18 @@ def assert_table_type(session: Session, table_name: str, table_type: str) -> Non
             expected_table_kind = table_type.upper()
         assert table_info[0]["kind"] == expected_table_kind
 
+    @staticmethod
+    def assert_rows_count(data: DataFrame, row_number: int):
+        my_iter = data.to_local_iterator()
+        row_counter = 0
+        for _ in my_iter:
+            len(data.schema.fields)
+            row_counter += 1
```

Comment:
Why not use `data.count()`? Otherwise this could be done more concisely like this:
```suggestion
        my_iter = data.to_local_iterator()
        row_count = sum(1 for _ in my_iter)
```

Reply:
Thanks @sfc-gh-jrose, I made the change.